### DISCUSSION TITLE
Make Quote reply button append text, not replace it

### DISCUSSION THREAD

**Discussion by KnorpelSenf** [community/community#2984] [Discussions] [answered]
When selecting a part of a post in a regular GH issue, the current behaviour of the Quote reply button is to append the selected text to the current input text field.

The new discussion feature provides a button with the same name, but when you click it, your complete text is erased and replaced by the text you were to quote. In addition, the editing history of the text field is cleared, so you cannot undo the operation. I'm not the only one who lost a long reply … I wish this was made similar to how it works in issues.

**Comment by becca** [comment, marked as answer]
Thank you for reporting this to us. This space is dedicated to feedback but we are definitely aware of this bug and are working to resolve it as soon as possible. We'll do our best to keep you posted on when it's resolved.

**Reply by becca** [reply]
This is now fixed
